haas/migrations/versions/c45f6a96dbe7_nic_primary_key_changed_to_bigint.py

```diff
@@ -0,0 +1,56 @@
+"""NIC primary key changed to BIGINT
+
+Revision ID: c45f6a96dbe7
+Revises: 3b2dab2e0d7d
+Create Date: 2017-06-14 10:36:17.744991
+
+"""
+
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = 'c45f6a96dbe7'
+down_revision = '3b2dab2e0d7d'
+branch_labels = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
```

Comment:
I think you can remove these comments that say "Autogenerated blah blah" because it's not really helpful.